code/modules/unit_tests/games/test_cards.dm

```diff
@@ -0,0 +1,36 @@
+/datum/unit_test/deck/Run()
+	return
+
+/datum/unit_test/deck/cards/Run()
+	return
+
+/datum/unit_test/deck/cards/proc/validate_deck(obj/item/deck/deck)
+	var/list/card_count = list()
+	for(var/datum/playingcard/card in deck.cards)
+		if(card_count[card.name] == null)
+			card_count[card.name] = 1
+		else if(card.name == "Joker")
+			card_count[card.name]++
+		else
+			// duplicate card in deck
+			return FALSE
+
+	if(length(card_count) != 53) // 2 Jokers, so 53 unique cards
+		return FALSE
+	return TRUE
+
+
+/datum/unit_test/deck/cards/initialized_properly/Run()
```

Comment:
I don't see for there to be any reason there is 3 unit tests when this could just be 1
```suggestion
/datum/unit_test/card_deck/Run()
```

Reply:
The idea was namespaced unit-tests, so for example later it could look like:

- unit_test/deck/cards/initialized_properly
- unit_test/deck/cards/dealing_cards
- unit_test/deck/cardhand/reveal

I agree it’s unnecessary to have 3 unit tests for this, but it was a side effect of this structuring. For now, it’s probably not needed and may be dropped, but is there any method for achieving this with DM, in-case it’s relevant later?